src/pocketmine/math/Vector3.php

```diff
@@ -158,6 +158,15 @@ public function getSide($side, $step = 1){
 	}
 
 	/**
+	 * Return a Vector3 instance
+	 * 
+	 * @return Vector3
+	 */
+	public function asVector3() : Vector3 {
+		return new Vector3($this->x, $this->y, $this->z);
```

Comment:
@thebigsmileXD this whole pull request adds functions that serve the exact same purpose as the `Player::getPosition()` method.